src/sentry/relay/config/__init__.py

```diff
@@ -241,7 +241,16 @@ def get_metrics_config(timeout: TimeChecker, project: Project) -> Mapping[str, A
     cardinality_limits: list[CardinalityLimit] = []
     for namespace in CARDINALITY_LIMIT_USE_CASES:
         timeout.check()
-        option = options.get(f"sentry-metrics.cardinality-limiter.limits.{namespace.value}.per-org")
+        if (
+            features.has("organizations:low-cardinality-limits-on-project", project.organization)
+            and namespace == UseCaseID.CUSTOM
+        ):
+            option = options.get("sentry-metrics.cardinality-limiter.limits.custom.per-org.low")
```

Comment:
You'll need to register this option in `sentry.options.default.

nit: Does it make sense to save a per-project limit in an option called "per-org"? I like the idea of putting the scope in the option, but maybe we should remove `per-org` from the option name.

Reply:
I agree, options is renamed and I have added it to defaults file